Portfolio/index.html

```diff
@@ -1,177 +1,205 @@
 <!DOCTYPE html>
-<html>

```

Comment:
You are missing the closing tag for the 'html' element. Add '</html>' at the end of the code.